peps/pep-0776.rst

```diff
@@ -0,0 +1,794 @@
+PEP: 776
+Title: Emscripten Support
+Author: Hood Chatham <roberthoodchatham at gmail.com>
+Sponsor: Łukasz Langa <lukasz at python.org>
+Discussions-To: https://discuss.python.org/t/84996
+Status: Draft
+Type: Standards Track
+Topic: Packaging
+Created: 18-Mar-2025
+Python-Version: 3.14
+Post-History: `18-Mar-2025 <https://discuss.python.org/t/84996>`__
+
+Abstract
+========
+
+`Emscripten <https://emscripten.org/>`__ is a complete open source compiler
+toolchain which compiles C/C++ code into WebAssembly/JavaScript executables that
+run in JavaScript runtimes including browsers and Node.js.
+
+This PEP formalizes the addition of Tier 3 for Emscripten support in Python 3.14
+which `was approved by the steering council on October 25, 2024
+<https://github.com/python/steering-council/issues/256>`__. The goal is to allow
+Pyodide wheels to be uploaded to PyPI.
+
+Motivation
+==========
+
+A web browser is a universal computing platform, available on Windows, macOS,
+Linux, and every smartphone.
+
+`The Pyodide project <https://pyodide.org/>`__ has for years supported
```

Comment:
```suggestion
`The Pyodide project <https://pyodide.org/>`__ has for several years supported
```

Reply:
Maybe we can be more specific: `has supported Emscripten Python since 2018`.